### PR TITLE
Adding Lambda to main

### PR DESCRIPTION
**Pull request by MinecraftSt3v3**

### CODE REVIEW

File: _sidebar.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 <!-- docs/_sidebar.md -->
 
+
+foo
```
Comment (zjorge96) [inline _sidebar.md:4]: What's this doing here? 🤔

File: _sidebar.md [hunk 2]
```diff
@@ -107,8 +108,7 @@
 - [7.4 - RBAC](7-kubernetes-container-orchestration/7.4-rbac.md)
 - [7.5 - HPAs](7-kubernetes-container-orchestration/7.5-hpas.md)
 - [7.6 - Webhooks](7-kubernetes-container-orchestration/7.6-webhooks.md)
-- [7.7 - K8s The Hard Way](7-kubernetes-container-orchestration/7.7-k8sthw.md)
-- [7.8 - Hello K8s](7-kubernetes-container-orchestration/7.8-hello-k8s.md)
+- [7.7 - Hello K8s](7-kubernetes-container-orchestration/7.7-hello-k8s.md)
```
Comment (zjorge96) [inline _sidebar.md:111]: This is going to remove 7.7 from the side bar.

File: _sidebar.md [hunk 3]
```diff
@@ -86,14 +90,11 @@
...
-  - [6.1.2 - Terraform State & Backends](6-infrastructure-configuration-management/6.1.2-terraform-backends.md)
-  - [6.1.3 - Terraform Modules](6-infrastructure-configuration-management/6.1.3-terraform-modules.md)
-  - [6.1.4 - Terraform Providers](6-infrastructure-configuration-management/6.1.4-terraform-providers.md)
-- [6.2 - Ansible](6-infrastructure-configuration-management/6.2-ansible.md)
-- [6.3 - Chef](6-infrastructure-configuration-management/6.3-chef.md)
+- [6.0 - Automation & Orchestration](6-automation-and-orchestration/6.0-overview.md)
+- [6.1 - Configuration Managers](6-automation-and-orchestration/6.1-configuration-managers.md)
+- [6.2 - Chef](6-automation-and-orchestration/6.2-chef.md)
+- [6.3 - Terraform](6-automation-and-orchestration/6.3-terraform)
+- [6.4 - Ansible](6-automation-and-orchestration/6.4-ansible.md)
```
Comment (zjorge96) [inline _sidebar.md:97]: This is following suit with the old sidebar configuration. Be sure to update it

**Review by zjorge96 — COMMENTED** [review]
Over all, the new lambda section looks good. The only small thing is that the side bar wasn't updated correctly.

File: _sidebar.md (hunk 3 above)
Comment (MinecraftSt3v3) [reply _sidebar.md]: Just updated 👍🏽

File: _sidebar.md [hunk 4]
```diff
@@ -40,8 +40,9 @@
   - [3.2.2 - EC2](3-cloud-computing/3.2.2-ec2.md)
   - [3.2.3 - Auto Scaling](3-cloud-computing/3.2.3-auto-scaling.md)
   - [3.2.4 - AWS with Packer](3-cloud-computing/3.2.4-aws-packer.md)
-  - [3.2.5 - ECS](3-cloud-computing/3.2.5-ecs.md)
-  - [3.2.6 - EKS](3-cloud-computing/3.2.6-eks.md)
+  - [3.2.5 - Lambda](3-cloud-computing/3.2.5-lambda.md)
+  - [3.2.6 - ECS](3-cloud-computing/3.2.5-ecs.md)
+  - [3.2.7 - EKS](3-cloud-computing/3.2.6-eks.md)
```
[the remark this answers is not in the document]
Comment (MinecraftSt3v3) [reply _sidebar.md:45]: I got this from main is that one not up to date?

**Review by zjorge96 — APPROVED** [review]
LGTM

**Pull request merged**